specs/_features/eip6110/beacon-chain.md

```diff
@@ -93,6 +93,7 @@ class ExecutionPayload(Container):
     withdrawals: List[Withdrawal, MAX_WITHDRAWALS_PER_PAYLOAD]
```

Comment:
the current tests are gated on EIP-6110 features and so I had to "forward port" the 4788 stuff to these documents